Makefile.am

```diff
@@ -268,6 +269,13 @@ tests_log_t_SOURCES = \
 	tests/log.cpp \
 	log.cpp
 
+tests_string_t_CPPFLAGS = $(THIRDPARTY_CPPFLAGS) @IRONMANCFLAGS@
+tests_string_t_LDADD = $(JSON_LIB) $(TAP_LIB)
+tests_string_t_SOURCES = \
```

Comment:
please add `tests/log_stub.cpp` in here; that will allow you to remove the `w_log` stub from the test case